### PR TITLE
Testing updates

### LINKED ISSUE #109: Define testing requirements for KGO-changing tickets
The WPs could make the testing requirements for KGO tickets more obvious. This should ensure that more developers run the right testing first time.

### LINKED ISSUE #183: Link or copy the local builds info for LFRic Apps to the Working Practices docs?
I was looking in the working practices docs on how to build locally and could only find info on running the test-suites for LFRic Apps.

Could the [local build wiki page](https://code.metoffice.gov.uk/trac/lfric_apps/wiki/local_builds) be linked or added to these working practice docs?

### PR DESCRIPTION
**Pull request by jennyhickson**
A few questions have come up around the KGO process, updating testing and KGO instructions here to try and clarify. Updating the LFRic Apps testing page to point to this and include the local_build.py info. 

Closes #183 
Closes #109

### CODE REVIEW

File: source/WorkingPractices/TestSuites/lfric_apps.rst [hunk 1]
```diff
@@ -3,14 +3,22 @@
...
 
-.. code-block::
+    .. code-block::
+
+        export CYLC_VERSION=8
+        rose stem --group=developer
+        cylc play <working copy name>
+        cylc gui
+
+Local testing:
```
Comment (richardgilham) [inline source/WorkingPractices/TestSuites/lfric_apps.rst:17]: Worth emphasising this is a basic command line-based test that doesn't use rose & cylc?

File: source/WorkingPractices/kgo.rst [hunk 2]
```diff
@@ -36,6 +21,62 @@ to this area is restricted to members of the Simulation Systems and Deployment
 Team and they will update the KGO files to include your new answers as part of
 the commit process.
 
+
+KGO Update Process
+------------------
+
+Getting the process right for KGO changing tickets significantly helps get
+such changes onto the trunk. When preparing your change for review:
```
Comment (richardgilham) [inline source/WorkingPractices/kgo.rst:29]: Worth saying (in the right place) that if you suspect a KGO change, run all as a precautionary measure?

**Review by richardgilham — CHANGES_REQUESTED** [review]
Very good.  A couple of suggestions for consideration.

File: source/WorkingPractices/TestSuites/lfric_apps.rst (hunk 1 above)
Comment (jennyhickson) [reply source/WorkingPractices/TestSuites/lfric_apps.rst:17]: done

File: source/WorkingPractices/kgo.rst (hunk 2 above)
Comment (jennyhickson) [reply source/WorkingPractices/kgo.rst:29]: I've added something in the "preparing for review" section - may or may not be the right space, but its a start

**Review by richardgilham — APPROVED** [review]
Great, thanks Jenny.

**Pull request merged**